Source/Core/Core/HW/BBA-TAP/TAP_Win32.cpp

```diff
@@ -242,99 +271,87 @@ bool CEXIETHERNET::IsActivated()
 
 bool CEXIETHERNET::SendFrame(u8 *frame, u32 size)
 {
-	DEBUG_LOG(SP1, "SendFrame %x\n%s",
-		size, ArrayToString(frame, size, 0x10).c_str());
-
-	OVERLAPPED overlap;
-	ZeroMemory(&overlap, sizeof(overlap));
+	DEBUG_LOG(SP1, "SendFrame %u\n%s", size, ArrayToString(frame, size, 0x10).c_str());
 
 	// WriteFile will always return false because the TAP handle is async
-	WriteFile(mHAdapter, frame, size, NULL, &overlap);
-
-	DWORD res = GetLastError();
-	if (res != ERROR_IO_PENDING)
+	if (!WriteFile(mHAdapter, frame, size, NULL, NULL))
 	{
-		ERROR_LOG(SP1, "Failed to send packet with error 0x%X", res);
+		DWORD err = GetLastError();
+		if (err != ERROR_IO_PENDING)
+		{
+			ERROR_LOG(SP1, "Failed to send packet with error 0x%X", err);
+		}
 	}
 
 	// Always report the packet as being sent successfully, even though it might be a lie
 	SendComplete();
-
+	INFO_LOG(SP1, "SendFrame complete");
```

Comment:
Logging is slow, so try not to introduce logs which are present in release builds in the send/recv paths. (iirc INFO_LOG is active in release, might be wrong)